model/src/main/java/org/cloudfoundry/identity/uaa/provider/AbstractExternalOAuthIdentityProviderDefinition.java

```diff
@@ -28,6 +28,7 @@
     private URL tokenUrl;
     private URL tokenKeyUrl;
     private String tokenKey;
+    private URL userInfoUrl;
```

Comment:
This parameter should be added to documentation, see
https://docs.cloudfoundry.org/api/uaa/version/75.2.0/index.html#oauth-oidc

Reply:
Sure! I had only given the canonical config for Github in `docs/github-oauth2-provider.md`.

All right, I've located the source of the slate API docs and pushed some documentation for this property!